tests/aio/test_tx.py

```diff
@@ -49,3 +49,45 @@ async def test_credentials():
         raised = True
 
     assert raised
+
+
+@pytest.mark.asyncio
+async def test_tx_snapshot_ro(driver, database):
+    session = await driver.table_client.session().create()
+    description = (
+        ydb.TableDescription()
+        .with_primary_keys("key")
+        .with_columns(
+            ydb.Column("key", ydb.OptionalType(ydb.PrimitiveType.Uint64)),
+            ydb.Column("value", ydb.OptionalType(ydb.PrimitiveType.Uint64)),
+        )
+    )
+    await session.create_table(database + "/test", description)
```

Comment:
It is better to delete table before create (it can execute many times for same database while development) or "test" table may be created in other test.

Reply:
good point, i'll fix it